src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -108,5 +113,32 @@ private static string LoadRules()
                     Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
                     "Rules/BuiltInRules.json"));
         }
+
+        /// <summary>
+        /// Loads the configurations file from default directory if no file was passed as an optional parameter.
+        /// </summary>
+        /// <param name="configurations">Optional Command-line Configurations paramater.</param>
+        /// <returns>Configurations file path if exists.</returns>
+        private string LoadConfiguration(string configurations)
+        {
+            try
+            {
+                if (configurations == null)
```

Comment:
This is expecting a file to be present by default. Is this expected?

Reply:
The intent was if a configurations param was not passed in w/ the cli, then see if there is one in the defaultPath to use. Does that make more sense?

Reply:
@anaismiller, I'm guessing the idea here is to always look to see if a user has defined a config at that path, and if they have, to go ahead and use it even if it wasn't specified, is that correct?  I'm guessing this because that file isn't part of this PR anywhere, so the user must define it on their own.

If the above is correct, this behavior needs to be documented, and I don't see that in the PR anywhere.  Can you please add that?  It should also at least be mentioned in the main README, with a link to the docs.